news/2023-01-01-osumonthly-6.md

```diff
@@ -0,0 +1,331 @@
+
+One thing worth pointing out is that the rules for the other two subforums, [Completed Skins](https://osu.ppy.sh/community/forums/109) and [Work In Progress](https://osu.ppy.sh/community/forums/119) skins, have remained the same, so please do keep note of that when you are submitting your skin.
+
+### Featured skins of the month
+
+Of course we did not port only the news section of the Skinners' Bulletin, but also the beloved skins of the month section in which we will continue to feature 3 skins every month, although we have cut down on its length. So without further ado lets get into this months skins.
+
+- [ALTITUDE](https://osu.ppy.sh/community/forums/topics/1684016): This skin contains a nice mix of glowy minimalism and grunge aesthetics, with great use of techniques commonly seen in UI design.
+- [io15](https://osu.ppy.sh/community/forums/topics/1688412): This skin based around the Hololive VTuber Airani Iofifteen manages to show off its theme quite well, by heavily incorporating design elements of the character, truly making it feel like the skin is dedicated to the characters.
+- [INGENIUM X](https://osu.ppy.sh/community/forums/topics/1697586): A skin with a minimalist approach inspired by Ingenium from My Hero Academia, with bold shapes and gradients mimicking the heat levels seen on thermal cameras.
+
+Check out a showcase of all of the skins below:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/videoseries?list=PLA2Nwtxa4WMqd2iI0wTjLUeWU8Gudhnuv" frameborder="0" allowfullscreen></iframe>
+
+If you want to get into skinning but don't know where to start, check out the [skinning tutorial](https://skinship.xyz/tutorial/introduction) or ask away on the [skinship Discord server](https://discord.skinship.xyz/)!
+
+## Did you know?
+
+Anyone's who remotely familiar with osu! is surely aware of [Mismagius'](https://osu.ppy.sh/users/19048) and [ztrot's](https://osu.ppy.sh/users/6347) [Can't Defeat Airman](https://osu.ppy.sh/beatmapsets/24313#osu/104229), one of osu!'s most played maps of all time. You might also know that there's another [Can't Defeat Airman](https://osu.ppy.sh/beatmapsets/24084#osu/82033) set by [Sushi](https://osu.ppy.sh/users/43108), but did you know that they were supposed to be a part of the same mapset?
+
+That's right, originally the map by Mismagius and ztrot were supposed to be a guest difficulty on Sushi's mapset, however due to multiple reasons, some of them being that the map was considered too hard to be ranked at the time (so it would have to be [approved](/wiki/Beatmap/Category), and Sushi felt like the guest difficulty would overshadow his entire mapset.
```

Comment:
```suggestion
That's right, originally the map by Mismagius and ztrot was supposed to be a guest difficulty on Sushi's beatmap. However, this didn't happen due to multiple reasons, some of them being that the map was considered too hard to be ranked at the time (so it would have to be [approved](/wiki/Beatmap/Category) instead), and Sushi feeling like the guest difficulty would overshadow his other difficulties.
```